types/pem/index.d.ts

```diff
@@ -244,8 +251,8 @@ export function createPkcs12(key: string, certificate: string, password: string,
  * @param callback Callback function with an error object and {pkcs12}
  * @returns the result of the callback
  */
-export function readPkcs12(bufferOrPath: string, options: Pkcs12ReadOptions, callback: Callback<{ pkcs12: any }>): any;
-export function readPkcs12(bufferOrPath: string, callback: Callback<{ pkcs12: any }>): any;
+export function readPkcs12(bufferOrPath: Buffer | string, options: Pkcs12ReadOptions, callback: Callback<{ pkcs12: any }>): Pkcs12ReadResult;
```

Comment:
`readPkcs12()` provides the result in the callback and not the result, so the only change here should be the type of `pkcs12` from `any` to `Pkcs12ReadResult`. The return type should be unchanged.

Reply:
thanks, fixed the return type for readPkcs12() and it's callback.